rvepak/converter.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+import string
+
+roman_map = {'I': 1, 'V': 5, 'X': 10, 'L': 50, 'C': 100, 'D': 500, 'M': 1000}
+
+
+class Converter(object):
+    def __init__(self, numerical, source_numeric_system,
+                 destination_numeric_system):
+        self.numerical = numerical
+        self.source_numeric_system = source_numeric_system
+        self.destination_numeric_system = destination_numeric_system
+
+    def roman_identifier(self):
+        for item in roman_map.keys():
+            if item in self.numerical:
+                return 1
+        else:
+            return 0
+
+    def roman_to_arabic(self):
+        if self.roman_identifier() == 1:
+            return sum([roman_map[x] for x in self.numerical])
+        else:
+            return 'Not a roman letter'
+
+    def arabic_to_roman(self):
+        if not self.numerical.isdigit():
+            print('{} is not a number'.format(self.numerical))
+        roman_output = ''
+        value_list = sorted(
+            {v: k
+             for k, v in roman_map.items()}.keys(), reverse=True)
+        print(value_list)
+        value_map = {v: k for k, v in roman_map.items()}
+        value = int(self.numerical)
+        for key in value_list:
+            if value % key != value:
+                roman_output += value_map[key] * int(value / key)
+                value = value % key
+        return roman_output
+
+
+convert = Converter('CCVII', 'Roman', 'Arabic')
```

Comment:
Something to consider going forward, is always name your variables when you are calling a function.  For example:
```
convert = Converter(numerical='CCVII', source_numeric_system='Roman', destination_numeric_system='Arabic')
````
This will allow other folks reading your code to know exactly what you are passing into a function, and how they might want to use it without having to reference the function itself and hunt for it in the code.